_posts/2021-02-19-status-update.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title: "2021-02-19: IAC Team Status Update"
+author: carabasdaniel
+categories:
+  - team
+  - status
+tags:
+  - community
+  - releases
+  - cloud-ci
+  - litmus
+  - PDKSync
+  - pdk-templates
+  - pdk
+---
+
+## Community Contributions
+
+We'd like to thank the following people in the Puppet Community for their contributions over this past week:
+
+- [`puppetlabs-apache#2122`][puppetlabs-apache-pr-2122]: "(IAC-1186) Add $use_port_for_filenames parameter", thanks to [smortex][smortex]
+- [`puppetlabs-apache#2120`][puppetlabs-apache-pr-2120]: "(MAINT) Use shared examples for facts + other spec cleanups", thanks to [ekohl][ekohl]
+- [`puppetlabs-java#461`][puppetlabs-java-pr-461]: "(MODULES-10935) - Switch legacy operatingsystem fact to modern kernel one", thanks to [rjd1][rjd1]
+- [`puppetlabs-kubernetes#473`][puppetlabs-kubernetes-pr-473]: "Fix calico CNI support", thanks to [djschaap][djschaap]
+- [`puppetlabs-mysql#1363`][puppetlabs-mysql-pr-1363]: "Support compression command and extension", thanks to [dploeger][dploeger]
+- [`pdk#941`][pdk-pr-941]: "Allow Facter4 to be co-installed with PDK", thanks to [GabrielNagy][GabrielNagy]
+- [`pdk#927`][pdk-pr-927]: "Adds AIX support when creating a new module", thanks to [logicminds][logicminds]
+- [`pdk-templates#400`][pdk-templates-pr-400]: "gitlab-ci: unbreak litmus acceptance tests", thanks to [cdenneen][cdenneen]
+- [`pdk-templates#399`][pdk-templates-pr-399]: "gitlab-ci: make litmus[variables] optional", thanks to [cdenneen][cdenneen]
+
+
+The IAC team would like to thank [Edwin Maldonado](https://github.com/mcka1n) from Puppets Solution Architect team, for joining our Community day this week. It was great having Edwin sit in and help out. 
+
+Edwin's first contribution during Community Day to [hiera-eyaml](https://github.com/voxpupuli/hiera-eyaml/pull/313) has already been released. 
+
+Check [all the ways to reach us]({% link _posts/2021-01-20-reaching-out.md %}) if you want to directly contact us about anything module related.
+
+## New Module Releases
+
+The following modules were released this week:
+
+- [`puppetlabs-apt`][puppetlabs-apt] (`7.7.1`)
+- [`puppetlabs-apache`][puppetlabs-apache] (`5.10.0`)
+- [`puppetlabs-mysql`][puppetlabs-mysql] (`10.10.0`)
+- [`puppetlabs-tagmail`][puppetlabs-tagmail] (`3.5.1`)
+
+  [puppetlabs-apt]: https://github.com/puppetlabs/puppetlabs-apt
+  [puppetlabs-apache]: https://github.com/puppetlabs/puppetlabs-apache
+  [puppetlabs-mysql]: http://github.com/puppetlabs/puppetlabs-mysql
+  [puppetlabs-tagmail]: https://github.com/puppetlabs/puppetlabs-tagmail/
+
```

Comment:
```suggestion

## Customer Defects Addressed

[MODULES-10899][MODULES-10899]: Thanks to [cdamage][cdamage] for liaising with the original reporter of this issue and getting a PR with a fix ready so promptly. 
We'd also like to thank [beatchristen][beatchristen] for additional contributions for this PR to allow PHP 8 packages to be installed on Debian 10 when using the experimental repositories.
Finally, thanks to [gguillotte][gguillotte] for reviewing and helping to get it over the line.
```

Links:
```md
[MODULES-10899]: https://tickets.puppetlabs.com/browse/MODULES-10899
```

Reply:
Added details as part of the Community Contributions